semperfiwebdesign/all-in-one-seo-pack
# remove rewrite_titles option #69

Issue #69 

## Proposed changes

Removes the Rewrite Titles options from General Settings.

## Types of changes

What types of changes does your code introduce?

- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Removing old code/functionality

## Checklist

- [x] I have added necessary documentation, or have created an issue for docs (if appropriate). (#2521)

## Testing instructions
- Make sure titles still get rewritten
- Peak at the code and look at what all functionality was touched by the checks for this option and make sure none of that functionality is broken
